### DISCUSSION TITLE
Auth by OTP phone

### DISCUSSION THREAD

**Discussion by alexdieudonne** [directus/directus#13981] [Feature Requests]
Is it possible to get a phone authentication or a custom way to get a token from API to authenticate user using custom way?

Different from `POST /auth/login`

**Comment by benhaynes** [comment]
I think it would be nice to have a magic link option that gets emailed... SMS would have to be an external integration, but I'd love to have a way to do that one too. There are likely other discussions for this already.

**Comment by anlexN** [comment]
when does directus support registering and logging in with phone numbers?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
